htdocs/inc.setPlayerBehaviourRFID.php

```diff
@@ -0,0 +1,73 @@
+
+<?php
+/*
+* GLOBAL settings for behaviour of placing or swiping RFID cards to
+* control player play and stop
+* SWIPENOTPLACE = Swiping starts the player
+* PLACENOTSWIPE = Placing the card starts player, removal stops it
+*/
+
+if(isset($_POST['rfidSwipePlace']) && trim($_POST['rfidSwipePlace']) != "") {
+    /*
+    * make sure it is a value we can accept, do not just write whatever is in the POST
+    */
+    if(trim($_POST['rfidSwipePlace']) == "SWIPENOTPLACE") {
+        $Swipe_or_Place = "SWIPENOTPLACE";
+        $exec = 'echo "'.$Swipe_or_Place.'" > '.$conf['settings_abs'].'/Swipe_or_Place';
+        if($debug == "true") {
+            print $exec; print "<br>Swipe_or_Place: ".$Swipe_or_Place;
+        }
+        exec($exec);
+    } elseif(trim($_POST['rfidSwipePlace']) == "PLACENOTSWIPE") {
+        $Swipe_or_Place = "PLACENOTSWIPE";
+        $exec = 'echo "'.$Swipe_or_Place.'" > '.$conf['settings_abs'].'/Swipe_or_Place';
+        if($debug == "true") {
+            print $exec; print "<br>Swipe_or_Place: ".$Swipe_or_Place;
+        }
+        exec($exec);
+    } 
+    // execute shell to create config file
+    $exec = "sudo ".$conf['scripts_abs']."/inc.writeGlobalConfig.sh";        
+    if($debug == "true") {
+        print $exec;
+    }
+    exec($exec);
```

Comment:
Please add an RFID Reader Service restart, so the settings change from the web app can be used instantly.
```
    // execute shell to restart RFID Reader Service
    $exec = "sudo systemctl restart phoniebox-rfid-reader.service";
    if($debug == "true") {
        print $exec;
    }
    exec($exec);
```

Reply:
@Toqqi i have added your required change